partials/tabs/tryonline.html

```diff
@@ -1,43 +1,205 @@
+        <div class="section-title blue-grey darken-4 white-text z-depth-1">
+          <span class="section-name">{{ section }}</span>
+          <span>
+            <a class="waves-light">
+              <i ng-click="add_bears(section)" class="fa fa-plus-circle white-text bears-icon" aria-hidden="true">
+              </i>
+            </a>
+          </span>
+          <span>
+            <a class="waves-light">
+              <i ng-click="remove_sections(section)" class="fa fa-minus-circle white-text bears-icon" aria-hidden="true">
+              </i>
+            </a>
+          </span>
+        </div>
         <div class="row">
-            <div class="progress center col m4 offset-m4">
-                <div class="indeterminate"></div>
+          <div class="col m3 s12">
+            <!-- Ignore HTML Linter because it only allows hyphens and lowercase in id attr -->
+             <!-- Ignore HTMLLintBear -->
+            <input type="text" id="sections[section]['files']" value="sections[section]['files']"
+            ng-model="sections[section]['files']" class="section-name">
+            <label for="sections[section]['files']">Files</label>
+          </div>
+        </div>
+        <div ng-repeat="(bear, val) in sections[section]['bears']" class="row">
+          <div class="col m3">
+            <div ng-if="bear.length>0" class="card-panel">{{bear}}</div>
+            <div ng-if="bear.length == 0" class="card-panel input-field">
+              <input type="text" id="autocomplete-input" class="autocomplete">
             </div>
+          </div>
+          <div class="col m8">
+            <ul class="left no-margin">
+              <li ng-repeat="(k,v) in val">
+                <div class="input-field no-margin">
+                <input type="text" ng-model="sections[section]['bears'][bear][k]">
+                <label>{{k}}</label>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="col m1">
+            <i ng-click="remove_bears(section, bear)" class="fa fa-minus-circle black-text bears-icon" aria-hidden="true">
+            </i>
+          </div>
+        </div>
+      </div>
+
+      <div class="row blue-grey lighten-5 black-text new-section z-depth-1">
+        <div class="col m3 offset-m3">
+          <input type="text" placeholder="Give a section name!" class="no-margins" id="new-section">
         </div>
-        <div class="center">Warming up the bears!</div>
+        <div class="col m6">
+            <a ng-click="add_sections()" class="btn waves-effect">
+              <span class="footer-btn-text">NEW SECTION</span>
+              <i class="fa fa-plus white-text bears-icon" aria-hidden="true"></i>
+            </a>
+        </div>
+      </div>
     </div>
+  </div>
+
+  <div ng-show="sections" class="row">
+    <div class="col m12 s12 center">
+      <div>
+        <a ng-click="run_coala()" class="center run-coala waves-effect waves-light btn">RUN 
```

Comment:
Trailing Whitespace: Trailing white spaces are unnecessary and can complicate diffs: Remove the ' ' at the end of the line.

*HTMLLintBear, severity MAJOR, section `html`.*